Watch Extension/FeedingView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct FeedingView: View {
+    @ObservedObject var store: Store<AppState, AppAction>
+    @State var isShowingActionSheet = false
+    let feeding: Feeding
+    var body: some View {
+        Text("00:00:00")
+            .actionSheet(isPresented: $isShowingActionSheet) {
+                ActionSheet(title: Text("What do you want to do?"),
+                            buttons: [
+                                .default(Text("Stop")) {
+                                    self.store.send(.stop(self.feeding))
+                                },
+                                // TODO: make this real
+                                .default(Text("Pause/Resume")) {
+                                    self.store.send(.pause(self.feeding))
+                                }
+                    ]
+                )
+        }
+        .gesture(TapGesture().onEnded {
+            self.isShowingActionSheet.toggle()
+        })
+    }
+}
+
+struct FeedingView_Previews: PreviewProvider {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/9397ad90d7ca569afad13609babe3778b69f0d95/Watch Extension/FeedingView.swift#L28"></span>Type name should only contain alphanumeric characters: ‘FeedingView_Previews’<br />
<code>type_name</code> <code>FeedingView.swift:28</code></td>
    </tr>
  </tbody>
</table>